idea_town/frontend/static-src/app/views/feedback-view.js

```diff
@@ -0,0 +1,94 @@
+import app from 'ampersand-app';
+import BaseView from './base-view';
+
+export default BaseView.extend({
+
+  _template: `
+    <div id="modal-screen">
+      <div id="{{id}}-modal" class="feedback-modal">
+        <h3 class="title">{{title}}</h3>
```

Comment:
I [had actually started with bindings & a collection](https://github.com/lmorchard/idea-town/blob/724aaffa417e0666651079f3de8aefa0fa7d9ad6/idea_town/frontend/static-src/app/views/enabled-feedback-view.js), but it seemed like overkill. The data never changes after the first render, and using mustache makes this little component a lot simpler, IMO.